.github/workflows/deploy-feed-discovery-image.yaml

```diff
@@ -0,0 +1,25 @@
+name: Build and push feed-discovery service to ghcr
+
+on:
+  push:
+    branches:
+      - 'issue-1743'
```

Comment:
I'm very interested in this GitHub Container Registry stuff. Were you able to push an image from your personal issue-1743 branch?

Reply:
Not yet, but having that allows the workflow to trigger when I push it to my fork for testing purposes.

Reply:
A couple if the iterations went through the entire process but failed on the final push, but it was using the wrong Dokerfile anyway. I figure if I can get it to do this on the proper file then when I switch that to master and merge it it *should* work 
![image](https://user-images.githubusercontent.com/59970326/151828524-165358ce-9603-4f6c-a3f7-9f457784c09c.png)